### ISSUE TITLE
Can we talk about quest gear? (not a coding issue)

### ISSUE THREAD

**Issue by dmhenderson76** [DarkCastleMUD/DarkCastle#25]
Here's the list as it currently stands, my comments in bold but first - a couple general statements re: what I think quest gear should be:

* Quest gear should be at least 1 step above the best gameload option out there for the class - this should account for set bonuses as well - **perhaps a quest set bonus by class** (2 pieces - exclude weapons/monk gloves) should be a thing?  Right now it's hard to steer folks away from the MOAD set - assuming there's no nerf there in the immediate future.

* Quest gear should offer something actually beneficial to the class that helps to define their role.  Ex: Useful:  A meaningful heroism boost on paladin quest gear.  Not useful: Batter/brace on barb gear.  

* People won't like this but quest gear should not be tradeable - even consider treating it like godload (can't scrap, can't be stolen but bound to the char).  It requires a significant time investment and paying someone a bunch of cash (which is ubiquitous now) to run it for you is kinda missing the point.

* Level req should remain 60.

* Strip +hp/mana/ki off quest gear.  It's not as big a deal in 2020.  Effects (unique ones especially) like the thief shadowslip cloak are king, add versatility and neat wrinkles to playing different classes

* Quest gear skill bonuses should be **meaningful and noticeable** and not a 1-2% bump (or less)

**Now for the sets themselves - my comments in bold:**

 ) The Embossed Leggings of Gireth hit -n- dam by 6, melee mitigation by 4, 
    hit_points by 100, onslaught by 6, [WARRIOR], costing 500 qpoints.

**Even with an extra +2 dam - no one would use this over the MOAD set unless they want +6 onslaught.**

2 ) Gaiot's Belt of Authority hit -n- dam by 6, melee mitigation by 4, hit_points by 100, combat mastery by 6, [WARRIOR], costing 500 qpoints.

**This one is fine.**

3 ) Berkthgar's Rage hit -n- dam by 6, hit_points by 100, hp regen by 5, 
    knockback by 6, [BARBARIAN], costing 500 qpoints.

**Why knockback over frenzy/rage/berserk?  Poor barbs this one is bad.  I would also argue that it might be bumped to 7/7 because barb**

4 ) Hades' Demonskin Boots hit -n- dam by 6, hit_points by 100, fly by 100, 
    batterbrace by 6, [BARBARIAN], costing 500 qpoints.

**Fly is great and useful.  I have literally never seen anyone ever use batter/brace and I bet I never will.  It's something that should be a level 30 skill, not 50+.  Swap for something useful**

5 ) Lu Bey's Curse hit -n- dam by 5, save_vs_acid by 6, hit_points by 100, 
    detect invisible by 100, sense life by 100, [THIEF], costing 500 qpoints.

**Amazing.  Beneficial to the class, helps them do stuff they can't normally do.  Perfect quest piece**

6 ) Vystir's Cape of Shadows hit -n- dam by 5, save_vs_poison by 6, 
    backstab by 6, escape by 6, [THIEF], costing 500 qpoints.

**Again, perfect piece.  Benefits thieves and what they do directly.  Better than all gameload.  No thief would ever run without this if given the choice**

7 ) a Crystal Orb Engulfed in Flames hit -n- dam by 5, 
    melee mitigation by 5, all saves by 6, armor by -30, solar gate by 6, spelldamage by 8, [MAGE], 
    costing 500 qpoints.

**Fine - but solar gate is not a mage's bread and butter.  I would make sure that a +6 to skill level is meaningful damage-wise before I say this is ok.  Increase skill bonus if it's not significant.  This is a recurring theme***

8 ) a Circle of Radiating Light invisibility by 100, reflect by 5, 
    spelldamage by 10, elemental filter by 6, [MAGE], costing 500 qpoints.

**Great.  No issue here.  Again, check the elemental filter spell calculation and make sure +6 is meaningful**

9 ) Drawan's Talisman hit -n- dam by 4, melee mitigation by 4, 
    spelldamage by 8, dispel magic by 6, [CLERIC], costing 500 qpoints

**MOAD elf head can go +8 SD/5 melee mit I believe.  This item should be at least +10.  Make sure dispel magic by 6 is meaningful and if not, increase skill bonus.**

10) an Elegant Ring of Flowers hit -n- dam by 3, 
    spelldamage by 8, melee mitigation by 4, iridescent aura by 6, [CLERIC], costing 500 qpoints.

**Good item.  Best cleric ring beneath this is wisp of smoke which can only go +7SD I believe.  Check skill bonuses to make sure increase is meaningful**

11) Armbands of Woven Yggradsil Roots hit -n- dam by 4, spelldamage by 8, 
    armor by -30, detect invisible by 100, colour spray by 6, [DRUID], costing 500 qpoints.

**Does AC actually do anything on DC?  I've heard rumors it does but never seen any definitive proof but regardless, how useful is -30 to a druid who shouldn't be tanking?   Replace with something else.  Druids don't get DI so that's nice.  Make sure color spray by 6 is meaningful.**

12) Strips of Knotted Bark hit -n- dam by 3, spelldamage by 8, mana by 100, 
    call lightning by 6, sun ray by 6, [DRUID], costing 500 qpoints.

**Ditch the +mana.  Check skills to make sure impact is meaningful, maybe add something else fun/unique**

13) The Gauntlets of Judgement hit -n- dam by 5, melee mitigation by 5, 
    mana regen by 5, divine fury by 6, hp regen by 5, spelldamage by 5, [PALADIN], costing 500 qpoints.

**Need to be +6 h/d as these are not arguably better than MOAD set (which go 5/5 + set bonus).  I can tell you for certain that +6 divine fury doesn't seem to do much at all.  Look at skill effectiveness**

14) Diamond-Plated Leggings hit -n- dam by 6, spell mitigation by 3, 
    melee mitigation by 3, song mitigation by 3, heroism by 6, spelldamage by 4, [PALADIN], costing 500 qpoints.

**Pretty sure +6 heroism does almost nothing due to the way it's calculated.  I get +23 to skills without leggings +25 with.  This sounds like it might be significant but if you take a look at (literally any) skill calculations, you'll see very quickly that you're talking about fractions of a single percentage point in terms of improvement.  This is actually what caused me to start looking at this**

15) The Soul of an Innocent hitroll by 5, damroll by 5, all saves by 4, 
    vampiric aura by 6, spelldamage by 4, [ANTI-PALADIN], costing 500 qpoints.

**Check skill boost from +6 vamp aura.  Rest is fine**

16) Marrok's Leggings of the Abyss hit -n- dam by 6, spelldamage by 6, 
    sneak by 100, reflect by 4, spell mitigation by 3, melee mitigation by 3, [ANTI-PALADIN], 
    costing 500 qpoints.

**These are fine if a bit uninteresting**

17) a Giant Eagle Feather hit -n- dam by 4, hit_points by 100, fly by 100, 
    archery by 10, [RANGER], costing 500 qpoints.

**This is godawful except for the fly.  It's outclassed by gameload and lol archery.  Ditch the +hp and add melee mit or something**

18) Montolio's Animal Skin Gloves hit -n- dam by 5, melee mitigation by 4, 
    mana by 100, offhand double by 6, herb lore by 6, [RANGER], costing 500 qpoints.

** Could argue MOAD gloves are better or equivalent minus the skill boosts.  Check skills to see if boost is meaningful**

19) The Bones of Istus hit -n- dam by 6, ki by 10, detect invisible by 100, 
    disrupt by 6, [MONK], costing 500 qpoints.

**Awesome - just make sure skill boosts are meaningful**

20) The Grandmaster's Hand Wraps hit -n- dam by 5, hit_points by 100, 
    punch by 6, meteor swarm by 3, [MONK], costing 500 qpoints.

**No one will ever use these since the OP thola-reshk shafts exist.  These cannot compete with a 4/4 +7 flamestrike +whatever poison held item where the monk can literally hold two - especially since the monk must choose between the held item and the gloves**

21) Blaine's Mask of Enchantment hit -n- dam by 5, melee mitigation by 5, 
    healing melody by 6, dodge by 4, [BARD], costing 500 qpoints.

**Bard gear with the same melee mit as war/pal pieces.  Ok.  Make sure skill boosts are useful but otherwise ok I guess.  Would swap out melee mit with something more appropriate for bards**

22) a Belt of Pink Satin hit -n- dam by 6, hit_points by 100, 
    melee mitigation by 4, armor by -30, tumbling by 6, [BARD], costing 500 qpoints.

**Another bard piece with as much melee mit as plate-wearing classes get.  Seems...wrong.  Make sure skill increases are blah blah**

23) Ghaerad, Sword of Legends 7d7, One-handed, hit -n- dam by 7, 
    wild magic by 6, flamestrike by 6, [WARRIOR PALADIN BARBARIAN RANGER], costing 500 qpoints.

**Ok, fuck this thing sideways.  The drawbacks to using this thing far outweigh any possible benefits it offers.  There are 1h gameload weapons that deal equivalent raw damage.  Add to that, it often refuses to let you remove it and can literally kill you.  The damage on wild magic spells is awful and meaningless and the potential BAD effects wild magic has (sanc opponent, FS opponent, TP opponent, heal opponent etc) are entirely too risky even if this thing had better dice than it does currently.  This thing is trash and although the what - 20 phrases(?) it can say were funny initially, it's literally just an annoyance and 75% of what it says won't even be funny to current players.  This item needs a major overhaul if anyone is ever going to use it again.  In order for someone to put up with this sword's bullshit, it needs to be far and away the best option for a 1h wield out there for the classes it's intended for**

24) Janikhel's Blade of Shadows 6d7, One-handed, hit -n- dam by 7, 
    vampiric touch by 3, weaken by 6, harm by 6, [THIEF ANTI-PALADIN BARD], costing 500 qpoints.

**Needs to be 7d7 or 7d8.  MA dagger can go 6d7 5/5 with custom flag.  Illusionary DoD is better.  Spell % are fine**

25) The Soul Grinder of Antioch 7d5, One-handed, hit -n- dam by 5, 
    spelldamage by 7, chill touch by 5, souldrain by 5, mana regen by 4, [MAGE CLERIC DRUID], 
    costing 500 qpoints.

**This one is fine**

26) Orro's Wand of Wonder costing 10 qpoints.

** No one knows what this does because no one will spend QP on a mystery item that has finite charges**

27) Orro's Special Brew costing 2 qpoints.

**I don't know what this does I'm sorry.  No one is going to spend QP on an item without knowing what it does (shop does not say)**

28) The Grandmaster's Cloak of the Ages hit -n- dam by 5, 
    melee mitigation by 4, sense life by 100, martial_defense by 5, [MONK], costing 500 qpoints.

**Fine I guess.  Does some nice stuff for monks.  Check skill level etc**

So anyway, just my two cents but I got to thinking about this after the heroism thing and talking to a barb and looking at his quest gear options and going "wow this really blows".  

**In summary:**
* More cool unique procs hidden in item descs (thief cloak)
* Strip off hp/mana/ki bonuses.  Chars that use these items won't need them
* Make sure that skill bonuses on quest gear actually matter
* Make skill choices not terrible.

**Comment by mcinelli61** [comment]
I've revised quest equipment for some classes so far, but not all. I agree
with getting rid of hp/mana/ki as most people wearing quest don't want
something on it that they can just meta. Without going through a piece by
piece itemization here, most of the pieces you marked as good are the ones
I've already revised. As far as conflicts with moad go, quest eq pre-dates
the revised moad equipment. If your class has a quest piece in a moad slot,
then you have a choice to make.

**Comment by dmhenderson76** [comment]
> . If your class has a quest piece in a moad slot, then you have a choice to make.

>  quest eq pre-dates the revised moad equipment

Understood (partially) but then quest gear needs to be looked at again if gameload is now eclipsing it - just my opinion. 

See I don't agree with the idea that someone should have to think about a choice between quest and gameload because quest stuff is supposed to be the best of the best, right?  IMO there shouldn't be anyone who looks at a quest piece and goes "nah I don't need that".   That's the point?  EQ stratification is a good thing!

Still have questions as to whether:
* +skill bonuses are **meaningful**.  Many do not appear to be looking at the code.  Again, talking less than a 1% skill bump in a lot of cases.  Seems like these should be higher.  Hell, the HOLY avenger has what, +20 behead and that's really not even statistically significant.  Most of these are like +6.

* Should there be a small class-specific set bonus for quest gear?  Might be a neat idea.  Tie it to signature skills?  Ex: Mages get +20 hellstream and +3 SD, thieves +10 dualbackstab and +5 hit, Paladins +20 behead and 2/2 or whatever.

* If there are going to be better gameload items than quest gear, then consider making quest gear !trade, !drop and !scrap like godload so it at least has a reason for folks to use it.

**Comment by TehDreadPirateRoberts** [comment]
I do agree with most of the original post here. Although I understand the comment regarding the choice between quest vs. Game load, there is no balance in these choices between the classes. Some classes currently don’t have to make that choice and end up with more hit/dam and melee mitigation than a plate wearing paladin or warrior. I think an adjustment to quest equ is a step towards many changes that need to take place to bring a little more balance to classes.

**Comment by bdunavant** [comment]
This is not a coding issue, and equipment is not even in the repo.   

This is a valid concern, and this discussion should be continued in the Dark Castle forum, or some other location appropriate for the content.
